src/libraries/System.Net.WebSockets.Client/tests/System.Net.WebSockets.Client.Tests.csproj

```diff
@@ -51,4 +41,10 @@
   <ItemGroup>
     <PackageReference Include="System.Net.TestData" Version="$(SystemNetTestDataVersion)" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetOS)' == 'Browser'">
+    <!-- TODO  
+    <ProjectReference Include="$(CommonTestPath)System/Net/Prerequisites/MonoNetTestServer/MonoNetTestServer.csproj" />
```

Comment:
1. I think you want `<ProjectReference ... ReferenceOutputAssembly="false"/>` because you just want to establish a build-order, not to actually link it in to the tests
2. You actually should reference a .proj that uses the `MSBuild` task (or maybe an Exec.  not sure) to build the server for the build machine RID, not for browser-wasm.